scripts/lib/CIME/utils.py

```diff
@@ -1787,3 +1787,7 @@ def string_in_list(_string, _list):
         if _string.lower() == x.lower():
             return x
     return None
+
+def model_log(model, arg_logger, msg):
```

Comment:
Okay - now I see why e3sm exclusively above - but this breaks our use of logger.debug to get these messages.  
I think that you may be able to fix it with
else:
  arg_logger.debug(msg) 
but that assumes that all of the messages replaced were debug messages.

Reply:
Yes, all the replaced calls were debug messages. I will see if I can restore these for you.